src/sp/it/pl/audio/tagging/Metadata.kt

```diff
@@ -74,9 +74,9 @@ private typealias F = JvmField
  *
  * The getters of this class return mostly string. For empty fields the output
  * is "" (empty string) and for non-string getters it varies, but it is never
- * null. See documentation
- * for the specific field. The rule is however, that the check for empty value
- * should be necessary only in rare cases.
+ * null. See documentation for the specific field. // TODO outdated?
```

Comment:
TODO is not a means to ask questions. Yes it is outdated. Pls remove the question mark. We want to not create a habit of abusing TODOs as a communication channel.

Reply:
yeah idk what that was